OpenBudgeteer.Blazor/Pages/Transaction.razor

```diff
@@ -183,8 +183,17 @@
                             </tr>
                         }
                     </table>
+                                    
+                    @if (transaction.Transaction.Amount != transaction.Buckets.Sum(b => b.Amount))
+                    {
+                        <div class="row" style="background-color: deepskyblue;  border-radius: 25px; margin: 0.3rem 0">
```

Comment:
If you want you can refactor `border-radius: 25px; margin: 0.3rem 0` to a seperate css class (together with line 213). I can also do it after the merge.